packages/mercury/src/base/_fields.scss

```diff
@@ -50,17 +56,23 @@
 /// @param {String} $field-block-selector [".field-block"] -
 /// @param {String} $field-inline-selector [".field-inline"] -
 /// @param {String} $field-inline__label-selector [".field-inline .label"] -
-/// @param {String} $field-group-cols-2-selector [".field-group-cols-2"] -
-/// @param {String} $field-group-cols-2__label-selector [".field-group-cols-2 .label"] -
+/// @param {String} $field-group-justified-start-selector [".field-group-justified-start"] -
+/// @param {String} $field-group-justified-start__label-selector [".field-group-justified-start .label"] -
+/// @param {String} $field-group-justified-end-selector [".field-group-justified-end"] -
+/// @param {String} $field-group-justified-end__label-selector [".field-group-justified-end .label"] -
 @mixin field(
   $field-group-selector: ".field-group",
   $field-group-inline-selector: ".field-group-inline",
   $field-selector: ".field",
   $field-block-selector: ".field-block",
   $field-inline-selector: ".field-inline",
   $field-inline__label-selector: ".field-inline > .label",
-  $field-group-cols-2-selector: ".field-group-cols-2",
-  $field-group-cols-2__label-selector: ".field-group-cols-2 .label"
+  $field-group-justified-start-selector: ".field-group-justified-start",
+  $field-group-justified-start__label-selector:
+    ".field-group-justified-start .label",
+  $field-group-justified-end-selector: ".field-group-justified-end",
+  $field-group-justified-end__label-selector:
+    ".field-group-justified-end .label"
```

Comment:
Why we can't use `.field-group-justified-start > .label`? We can make the selector to fail sooner by adding `>`, which will reduce the rendering time of the page.